### ISSUE TITLE
Fix ISS notebook registration API

### ISSUE THREAD

**Issue by ambrosejcarr** [chanzuckerberg/starfish#90]
* putting this here so I remember. 

ISS notebook is currently does not work because the registration module has moved. 

```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-8-7b1a2d078906> in <module>()
----> 1 from starfish.registration._fourier_shift import FourierShiftRegistration
      2 from starfish.registration import Registration
      3 
      4 upsample = 1000
      5 s = Registration.run("FourierShiftRegistration", s, upsample)

ModuleNotFoundError: No module named 'starfish.registration'
```

When I fix that, the pipeline appears to have changed the run API, so either @ttung can take a stab at it or I'll review the commits that changed this when I've had more sleep. 

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-8-19c5ac120260> in <module>()
      3 
      4 upsample = 1000
----> 5 s = Registration.run("FourierShiftRegistration", s, upsample)

/usr/local/lib/python3.6/site-packages/starfish/pipeline/pipelinecomponent.py in run(cls, algorithm_name, stack, *args, **kwargs)
     20     def run(cls, algorithm_name, stack, *args, **kwargs):
     21         """Runs the registration component using the algorithm name, stack, and arguments for the specific algorithm."""
---> 22         algorithm_cls = cls._class_for_algorithm(algorithm_name)
     23         instance = algorithm_cls(*args, **kwargs)
     24         return instance.register(stack)

AttributeError: type object 'Registration' has no attribute '_class_for_algorithm'
```

**Comment by ttung** [comment]
This is why we need #81 landed (blocked on #80).

**Comment by ttung** [comment]
Oh wait, this wasn't ever converted to .py.  That's why I missed this.

**Comment by ambrosejcarr** [comment]
Yep, that's right. Sorry if I wasn't clear--this wasn't converted and therefore was missed in the refactor.

**Comment by ambrosejcarr** [comment]
Fixed in #92
